envs/example/defaults.yml

```diff
@@ -517,9 +564,13 @@ percona:
 common:
   ipmi:
     enabled: False
-    state: probe
   system_tools:
-    mcelog: False
+    mcelog:
+      enabled: False
```

Comment:
woops.  weird that it worked ... I guess mcelog being a dict was enough to fail the bool test on this.